feature/google/src/main/java/com/revenuecat/purchases/google/GoogleStoreProduct.kt

```diff
@@ -20,6 +24,16 @@ data class GoogleStoreProduct(
     val productDetails: @RawValue ProductDetails // TODO parcelize?
 ) : StoreProduct, Parcelable {
 
+    override val purchasingData: PurchasingData
+        get() = if (type == ProductType.SUBS && defaultOption != null) {
+            defaultOption.purchasingData
```

Comment:
i suppose we'd never have defaultOption == null if type == SUBS...but maybe worth having a branch + error for that?